includes/options/views/class-amp-setup-wizard-submenu-page.php

```diff
@@ -0,0 +1,223 @@
+<?php
+/**
+ * AMP setup wizard page.
+ *
+ * @package AMP
+ * @since @todo NEW_ONBOARDING_RELEASE_VERSION
+ */
+
+/**
+ * AMP setup wizard submenu page class.
+ *
+ * @since @todo NEW_ONBOARDING_RELEASE_VERSION
+ */
+class AMP_Setup_Wizard_Submenu_Page {
+	/**
+	 * Handle for JS file.
+	 *
+	 * @var string
+	 */
+	const JS_HANDLE = 'amp-setup';
+
+	/**
+	 * HTML ID for the app root element.
+	 *
+	 * @var string
+	 */
+	const APP_ROOT_ID = 'amp-setup';
+
+	/**
+	 * The minimum core WP version whose core assets are compatibile with the setup feature.
+	 *
+	 * @var float
+	 */
+	const TARGET_WP_VERSION = 5.4;
```

Comment:
Using floating point numbers to make exact comparisons is prone to error and should be avoided.

The version, as interpreted by floating point math depending on the architecture it runs on, could turn something like `5.4` into `5.3999999..`, throwing everything off.

This should use a string version and use `version_compare()` instead.
```suggestion
	/**
	 * The minimum core WP version whose core assets are compatibile with the setup feature.
	 *
	 * @var string
	 */
	const TARGET_WP_VERSION = '5.4';
```